defaults/main.yml

```diff
@@ -465,7 +465,14 @@ apache__combined_snippets: '{{ apache__dependent_snippets
 apache__https_enabled: '{{ ansible_local|d() and ansible_local.pki|d() and
                            (ansible_local.pki.enabled|d() | bool) and
                            apache__https_listen|length > 0 }}'
+                                                                   # ]]]
+
```

Comment:
There is a linting tool available called [yaml4rst](https://github.com/ypid/yaml4rst) which implements the current format. Can you give it a try?